src/Swashbuckle.AspNetCore.SwaggerGen/SchemaGenerator/MemberInfoExtensions.cs

```diff
@@ -2,22 +2,22 @@
 using System.Linq;
 using System.Reflection;
 using Microsoft.AspNetCore.Mvc;
-
+using System;
 namespace Swashbuckle.AspNetCore.SwaggerGen
 {
     public static class MemberInfoExtensions
     {
         public static IEnumerable<object> GetInlineOrMetadataTypeAttributes(this MemberInfo memberInfo)
         {
-            var metadataTypeAttribute = memberInfo.DeclaringType.GetCustomAttributes(true)
+            var metadataTypeAttribute = memberInfo?.DeclaringType.GetCustomAttributes(true)
                 .OfType<ModelMetadataTypeAttribute>()
                 .FirstOrDefault();
 
-            var metadataTypeMemberInfo = metadataTypeAttribute?.MetadataType.GetMember(memberInfo.Name)
-                .FirstOrDefault();
+            var metadataTypeMemberInfo = metadataTypeAttribute?.MetadataType.GetMember(memberInfo?.Name)
+                ?.FirstOrDefault();
```

Comment:
I'm fairly certain these additions are all redundant

Reply:
My pref is to avoid null checking the extended instance (i.e. `memberInfo`) from within the extension method. I prefer this responsibility to be on the caller as is the case with regular instance methods.